### PR TITLE
Qt5→Qt6

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-`libqtxdg` is a Qt 5 implementation of freedesktop.org XDG specifications.   
+`libqtxdg` is a Qt 6 implementation of freedesktop.org XDG specifications.
```
Comment (tsujan) [inline README.md:5]: What about "... is a Qt implementation of..."?

Comment (stefonarch) [reply README.md:5]: Preparing for Qt 7, why not.  It came to my mind that we could add instructions for building Qt5 (with `checkout <commit>` ?).

Comment (tsujan) [reply README.md:5]: The Qt version is obvious from the dependencies. The main info here is that libqtxdg is a *Qt implementation* of XDG specs.

**Comment by tsujan** [comment]
Done in https://github.com/lxqt/libqtxdg/pull/301

**Pull request closed without merging**